### DISCUSSION TITLE
Tasmota Device Group with 2 Treatlife DS03

### DISCUSSION THREAD

**Discussion by maranhaoBruno** [arendst/Tasmota#18003] [Q&A]
I have two Treatlife DS03's that I recently purchased and removed the stock chip and replaced them with ESP8266's flashed with Tasmota. The devices were set up as described here (https://templates.blakadder.com/treatlife_DS03.html) and here (https://digiblur.com/2020/07/03/the-treatlife-esmlfe-tasmota-fan-controller/).

One of the devices is connected to the ceiling fan with lights, and the other is just connected to line power with the fan and light loads capped. I would like them to operate as a three way switch such that changes made to either switch are reflected to the other, and as such will effect the fan and light.

I thought I would do this with the device groups feature. However, the only commands that seem to be shared between the switches is the on/off status of the fan and light. The fan speed and the light dimness level are not shared. Any help in troubleshooting and resolving this would be much appreciated.

**Comment by ferbulous** [comment]
Hi, did you manage to figure this out?
